definitions/tools/vep.cwl

```diff
@@ -68,15 +61,10 @@ inputs:
             valueFrom: |
                 ${
                     if (inputs.custom_gnomad_vcf) {
-                        return ['--check_existing', '--custom', inputs.custom_gnomad_vcf.path + ',gnomADe,vcf,exact,0,AF,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_SAS']
+                        return ['--check_existing', '--custom', inputs.custom_gnomad_vcf.path + ',custom_gnomAD,vcf,exact,0,AF,AF_AFR,AF_AMR,AF_ASJ,AF_EAS,AF_FIN,AF_NFE,AF_OTH,AF_SAS']
```

Comment:
Does this means we need to change this filter?
https://github.com/genome/analysis-workflows/blob/de81bd20ce8829ef64146115902a10e4fe7bad0b/definitions/tools/filter_vcf_gnomADe_allele_freq.cwl#L31

Reply:
I would suggest leaving the custom tag as `gnomADe`.

 1. we don't have to update the filter
 2. the string `gnomADe` denotes it's from the "exome" and `gnomADw` would be "Whole-genome"
 3. In general I'd suggest keeping VCF tags as short as possible. The header contains additional information including a description. As long as the tag is a unique string.

Reply:
I don't quite understand how the custom VCF works. Will this include the `genomADe` string the in the VEP tags or are the `AF` etc tags used plainly and would, thus, conflict with the AF tag written with the `--everything` flag.